backend/hitas/views/indices.py

```diff
@@ -93,7 +93,7 @@ def get_object(self):
             return self.model_class(month=self.kwargs["month"])
 
     def get_queryset(self):
-        return self.model_class.objects.only("month", "value").all().order_by("-month")
+        return self.model_class.objects.filter(value__gt=0).only("month", "value").all().order_by("-month")
```

Comment:
no need for filter when value is constrained to be `>= 0`?

Reply:
My thinking was that there still may be old `0` value indices in the db. Instead of this i will add a migration to completely remove those.

Reply:
there might. I always forget that `validators` do not add the database constraints automatically :cry: It requires a manual
```
constraints = [
    models.CheckConstraint(name="index_value_positive", check=models.Q(value__gte=0)),
]
```

these would be nice to have to enforce consistency between code and the database